docs/mine-hnt/data-only-guides/dragino-pg1301.mdx

```diff
@@ -0,0 +1,284 @@
+   Oct 02 15:07:19 raspi helium_gateway[917]: starting, oui: 2, uri: http://54.176.88.149:8080/, public_key: 1124CJ9yJaHq4D6ugyPCDnSBzQik61C1BqD9VMh1vsUmjwt16HNB, module: router
+   Oct 02 15:07:19 raspi helium_gateway[917]: starting, oui: 1, uri: http://52.8.80.146:8080/, public_key: 112qB3YaH5bZkCnKA5uRH7tBtGNv2Y5B4smv1jsmvGUzgKT71QpE, module: router
+   Oct 02 15:07:27 raspi helium_gateway[917]: new packet forwarder client: MacAddress(DC:A6:32:8B:F6:6F:FF), 127.0.0.1:44003, module: gateway
+   ```
+6. If the service is not running, start it with
+   `sudo systemctl restart helium_gateway`
+7. To enable the helium gateway service to start at boot time, run
+   `sudo systemctl enable helium_gateway` . If the service is already enabled at
+   boot, the command will return no output.
+
+Congradulations, your LoRa Packet Forwarder and Helium Gateway service is setup!
+At this point your LoRa radio will be able to pass radio traffic between the
+network and the Helium gateway. Any devices you registered via the Helium
+console should now be live. Your hotspot will not appear on the map. Before
+adding your hotspot to the blockchain, please check and backup your
+configuration.
+
+## Check and Backup your Configuration
+
+1. Run `helium_gateway key info` to view your gateway address and unique animal
+   name assigned to you. Example:
+   ```
+   $ helium_gateway key info
+   {
+   "address": "13XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+   "name": "unique-animal-name"
+   }
+   ```
+2. Backup your gateway key to another machine for recovery purposes. The file
+   `/etc/helium_gateway/gateway_key.bin` contains the key.
+
+:::warning If you lose this key, you will not be able to use this hotspot
+anymore in case there is a problem on your hostpot (SD card corrupted, etc). You
+will need to generate a new key and lose your existing animal name/hostpot
+configuration :::
+
+## Testing the packet transfer with an end device
+
+If you have an end device (LoRaWAN node), you can check if the hotspot can
+transfer packets through the network. In other words, you can check if the
+devices can send data to the Helium Console. The Helium Console is where you can
+provision and manage the end devices, so the nodes need to be registered to
+start using the network.
+
+### Some comments and recommendations about the Helium Console
+
+At least for now, it is recommended to use the staging console instead of the
+production one when we are transferring data through Data Only hotspots. You can
+differentiate them for the URL they use,
+[console.helium.com](https://console.helium.com/) for production, and
+[staging-console.helium.wtf](https://staging-console.helium.wtf) for staging.
+Don't be surprised if you find some differences between those console flavors,
+basically staging is where the new features are tested before their rollout to
+production.
+
+:::important If the device you will use for testing is registered on the
```

Comment:
These tags should be on their own line alone.

:::important
text
:::

Reply:
fixed